guides/common/modules/proc_preparing-for-using-external-databases.adoc

```diff
@@ -0,0 +1,27 @@
+[id="Preparing_for_using_external_databases_{context}"]
+= Optional: Preparing {Project} for using external databases
+
+By default, the {Project} installation process includes installing a PostgreSQL database on the same host as {ProjectServer}.
+However, in certain {Project} deployments, using external databases instead of the default local databases can help with the server load.
+
+You can set up your {Project} deployment to use an external PostgreSQL database during the installation of {Project}. 
+You can also migrate to an external PostgreSQL database on an existing {Project} deployment.
+
+.Procedure
+
+. Consider whether using an external database is beneficial for your use case. 
+For more information, see {AdministeringDocURL}postgresql-as-an-external-database-considerations_admin[PostgreSQL as an external database considerations] in _{AdministeringDocTitle}_.
+
+. If you want to use an external external PostgreSQL database for your {Project} deployment, prepare your {Project} environment as follows:
```

Comment:
```suggestion
. If you want to use an external PostgreSQL database for your {Project} deployment, prepare your {Project} environment as follows:
```

Reply:
But actually, if you turn the previous step into a prerequisite, you should be able to drop this line and then only keep lines 17-21 as the actual steps. It would make the procedure shorter and simpler, which is a big plus IMO.